modules/nw-network-flows-object.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/tracking-network-flows.adoc
+
+[id="nw-network-flows-object_{context}"]
+= Network object configuration for tracking network flows
+
+The following YAML shows the configuration fields for network flows collectors in the Cluster Network Operator configuration.
+
```

Comment:
Suggested edit: I would recommend adding a `.Example YAML title` to the example file

Reply:
I agree that I should have used a title, but this YAML forest is replaced in favor of a table.